_includes/v19.2/sql/movr-start.md

```diff
@@ -0,0 +1,6 @@
+- Run [`cockroach demo movr`](cockroach-demo.html) to open an interactive SQL shell to a temporary, in-memory cluster with the `movr` database preloaded and set as the [current database](sql-name-resolution.html#current-database).
+- Use [`cockroach workload`](cockroach-workload.html):
+ 1. Start up a [secure](secure-a-cluster.html) or [insecure](start-a-local-cluster.html) local cluster.
+ 1. Run `cockroach workload init movr` with the appropriate flags and [connection string](connection-parameters.html) to initialize and populate the `movr` database on your running cluster.
```

Comment:
Could we show the exact command string here for at least the simplest case?  And keep the links for people who need something more complicated

Reply:
Sure thing!